content/dart/concepts/queue/queue.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: 'Queue'
+Description: 'A collection that can be manipulated at both ends.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Classes'
+  - 'Queues'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+A **Queue** is a collection that can be manipulated at both ends. A Queue inserts data from one end and deletes data from another end, it is considered a "first in first out" collection.
+
+## Syntax
+
+Creating a queue:
+
+```
+Queue variable_name = new Queue();
+```
+You can also create a queue from an existing list:
+
+```
+var variable_name = new Queue.from(list_name);
+```
+It must be noted that to use a queue in a dart program you have to import "dart:collection" module. If you don’t do so then you will see the following error:
```

Comment:
Give one line space before and after a code block. You can add this line as a note. Here's an example of a note - 
> **Note:** Note Text.

Reply:
Hello, I commited all the other changes except this one. I'm not really sure what this means, do you want me to add an empty note line before and after each code block?